Sources/SwiftParser/TokenConsumer.swift

```diff
@@ -82,6 +82,24 @@ extension TokenConsumer {
     }
   }
 
+  @inline(__always)
+  mutating func at(
+    _ spec1: TokenSpec,
+    _ spec2: TokenSpec,
+    _ spec3: TokenSpec,
+    _ spec4: TokenSpec,
+    _ spec5: TokenSpec
+  ) -> Bool {
+    switch self.currentToken {
+    case spec1: return true
+    case spec2: return true
+    case spec3: return true
+    case spec4: return true
+    case spec5: return true
+    default: return false
+    }
+  }
```

Comment:
I’d like to avoid spelling out too many copies of this function. What I’ve done before is use `at(spec1, spec2, spec3) || at(spec4, spec5)` or (preferred) create a `TokenSpecSet` for the five specs. I think in your case it would make sense to create a `TokenSpecSet` for the variable introducers.